projects/packages/videopress/src/client/admin/components/video-card/index.tsx

```diff
@@ -63,10 +77,16 @@ export const VideoCard = ( {
 	editable,
 	showQuickActions = true,
 	loading = false,
+	uploading = false,
+	processing = false,
 	onVideoDetailsClick,
 }: VideoCardProps ) => {
-	const isBlank = ! title && ! duration && ! plays && ! thumbnail;
+	const isBlank = ! title && ! duration && ! plays && ! thumbnail && ! loading;
+
+	// Mapping thumbnail (Ordered by priority)
 	thumbnail = loading ? <Placeholder width={ 360 } /> : thumbnail;
+	thumbnail = uploading ? <UploadingThumbnail /> : thumbnail;
+	thumbnail = processing ? <ProcessingThumbnail /> : thumbnail;
```

Comment:
I think we should not use `thumbnail` directly here, as that is the _prop_

Reply:
Done: https://github.com/Automattic/jetpack/pull/26627/commits/491d932b148c94d552deb0c33fdd11f68ce6ba58